internal/download/artifact.go

```diff
@@ -32,3 +36,43 @@ func Cleanup(directory string) {
 		log.Err(err).Msg("Unable to cleanup previous artifacts")
 	}
 }
+
+// GetDirName returns a target folder name
```

Comment:
That's quite the simple explanation for what's actually going on here 😅 
Might want to expand a little bit on the detail.